src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -573,9 +573,9 @@ namespace GraphQL.Builders
             where TArgumentGraphType : GraphQL.Types.IGraphType { }
         public virtual GraphQL.Builders.ConnectionBuilder<TSourceType> Argument<TArgumentGraphType>(string name, string? description, System.Action<GraphQL.Types.QueryArgument>? configure)
             where TArgumentGraphType : GraphQL.Types.IGraphType { }
-        public virtual GraphQL.Builders.ConnectionBuilder<TSourceType> Argument<TArgumentGraphType, TArgumentType>(string name, string? description, [System.Diagnostics.CodeAnalysis.AllowNull] TArgumentType defaultValue = default)
+        public virtual GraphQL.Builders.ConnectionBuilder<TSourceType> Argument<TArgumentGraphType, TArgumentType>(string name, string? description, TArgumentType defaultValue = default)
             where TArgumentGraphType : GraphQL.Types.IGraphType { }
-        public virtual GraphQL.Builders.ConnectionBuilder<TSourceType> Argument<TArgumentGraphType, TArgumentType>(string name, string? description, [System.Diagnostics.CodeAnalysis.AllowNull] TArgumentType defaultValue, System.Action<GraphQL.Types.QueryArgument>? configure)
```

Comment:
.NET 6 TFM side effect ? I think that in case of ifdefs there should be multiple approval files - for each TFM.

Reply:
I wouldn’t. It will just make it more painful to update the approval files. And then whenever there is an api change, there will be three approval files to review. The approvals are supposed to help us, not annoy us.

Reply:
First of all, it should reflect reality. Now it shows public API for NET6 only. As soon as we have different APIs for different TFM, we actually **have to** have different approval files. 

> The approvals are supposed to help us, not annoy us.

Agree. That is why we need to make different approval files in order not to get confused in subsequent independent changes for netstandard/NET TFMs.

Reply:
No, we don't.  The existing approval files are sufficient.  There is no confusion.  Nobody looks at the approval files to know what APIs are supported.  It's just a way for us to review changes to the public API base to be sure we don't issue a breaking change.  It's not a published document that shows that for such-and-such a TFM there are specific public members.

Reply:
I did not say that this is a public document.

>  Nobody looks at the approval files to know what APIs are supported for a specific TFM.

**We** look to know about breaking changes. It's for us of course. Imagine one adds `TimeOnlyGraphType` in netstandard2.0 TFM in v4.8. Then for some reason this type will be removed in v4.9. It's a breaking change obviously but we will not even notice this change, it will be hidden from our eyes because now we see only public API changes for NET6.

Reply:
I do not think this "issue" is worth the considerable trouble to implement.

Reply:
I will try.

Reply:
#2730

Reply:
Perhaps you misunderstood me.  I was not talking about the effort to implement multiple approval files.  I was talking about the effort keep them up to date and review them on an ongoing basis in the future.

Reply:
if we were using Verify i would use a  `UniqueFor`  to produce one approved file per 
 target fwk https://github.com/VerifyTests/Verify/blob/main/docs/naming.md#uniquefor 

> Perhaps you misunderstood me. I was not talking about the effort to implement multiple approval files. I was talking about the effort keep them up to date and review them on an ongoing basis in the future.

verify supports bulk approval (through a system wide kb shortcut or a UI)

Reply:
but IMO this topic should not block this PR. i can be handled in a diff issue/pr

Reply:
I fixed approval files bloat and `AllowNull` attributes side effect as well. See #2730 again, please.

Reply:
I suggest merge this PR after #2730

Reply:
OK, this goes first :)